src/main/scala/path/Relationship.scala

```diff
@@ -26,7 +27,17 @@ sealed trait RelationshipType extends ToQueryWithIdentifiers with Referenceable
     else ":" + (kinds mkString "|")
 }
 
-class AnyRelationship() extends RelationshipType {
+class CreateRelationship(val kind: String, val properties: Seq[Property]) extends Relationship {
+  def toQuery(referenceableMap: ReferenceableMap): String =
+    makeRelationshipString(referenceableMap, Seq(kind), Property.toQuery(properties))
+}
+
+object CreateRelationship {
+  def apply(kind: String, properties: Seq[Property] = Seq.empty): CreateRelationship =
+    new CreateRelationship(kind, properties)
+}
+
+class AnyRelationship() extends Relationship {
```

Comment:
nope, because i use references to lookup identifiers. by design, every time you create a new node/relationship, it has to be a distinct object so that identifiers can be assigned appropriately. 

this is also why i am using classes over case classes and manually defining `apply`... in this world:
```scala
val x = KindRelationship("label1")
val y = KindRelationship("label1")
x != y
```